kas/iot2050.yml

```diff
@@ -30,6 +30,9 @@ repos:
     refspec: cf3f18a649ff42726677a58b224b9bb58ea4ba46
     layers:
       meta:
+    patches:
+      dynamic-load-dtbo:
+        path: isar-patches/0001-update-uboot-script-dynamic-load-the-dtbo-file.patch
```

Comment:
You should use https://github.com/ilbers/isar/commit/8b9f930410234ad46135df3642bc0c7053fc854b by now.

Reply:
Have referenced this commit, but I haven't started testing yet, don't know if it will work